jujugui/static/gui/src/app/maraca/prop-types.js

```diff
@@ -0,0 +1,150 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const PropTypes = require('prop-types');
+const shapeup = require('shapeup');
+
+let propTypes = {};
+
```

Comment:
It would be nice if there was a comment on each one of these proptypes outlining what they are.